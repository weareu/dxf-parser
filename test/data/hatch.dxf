0
SECTION
2
ENTITIES
0
HATCH
5
20000
8
0
100
AcDbEntity
100
AcDbHatch
10
0
20
0
30
0
210
0
220
0
230
1
2
SOLID
70
1
71
0
91
1
92
2
72
0
73
1
93
6
10
481167.286973555
20
6757868.46229187
10
483012.242899119
20
6758349.75514202
10
482771.596474045
20
6754740.05876591
10
479964.054848188
20
6752814.88736533
10
477156.513222331
20
6755782.85994123
10
481167.286973555
20
6757868.46229187
97
0
75
0
76
1
98
0
0
HATCH
5
20001
8
0
100
AcDbEntity
100
AcDbHatch
10
0
20
0
30
0
210
0
220
0
230
1
2
SOLID
70
1
71
0
91
1
92
2
72
0
73
1
93
19
10
478259.476003918
20
6765308.44760039
10
477697.967678746
20
6763062.4142997
10
476534.843290891
20
6763744.24583741
10
477256.782566112
20
6765669.417238
10
483914.666993144
20
6766591.89520078
10
487724.902056807
20
6765990.2791381
10
487765.009794319
20
6761297.67384917
10
484797.037218413
20
6764987.58570029
10
482029.603330069
20
6765629.30950049
10
481909.280117532
20
6762059.7208619
10
480385.186092067
20
6759573.04113614
10
475411.826640548
20
6758008.83937316
10
467951.787463271
20
6760455.41136141
10
466908.986287953
20
6764746.93927522
10
474770.102840353
20
6761337.78158668
10
478379.799216454
20
6760335.08814887
10
479984.108716944
20
6762661.33692458
10
480746.155729677
20
6764787.04701273
10
478259.476003918
20
6765308.44760039
97
0
75
0
76
1
98
0
0
ENDSEC
0
EOF
