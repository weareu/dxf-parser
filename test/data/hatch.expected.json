{
	"entities": [
		{
			"type": "HATCH",
			"boundaries": [
				[
					{
						"x": 481167.286973555,
						"y": 6757868.46229187
					},
					{
						"x": 483012.242899119,
						"y": 6758349.75514202
					},
					{
						"x": 482771.596474045,
						"y": 6754740.05876591
					},
					{
						"x": 479964.054848188,
						"y": 6752814.88736533
					},
					{
						"x": 477156.513222331,
						"y": 6755782.85994123
					},
					{
						"x": 481167.286973555,
						"y": 6757868.46229187
					}
				]
			],
			"handle": "20000",
			"layer": "0",
			"elevationX": 0,
			"elevationY": 0,
			"elevationZ": 0,
			"extrusionDirectionX": 0,
			"extrusionDirectionY": 0,
			"extrusionDirectionZ": 1,
			"patternName": "SOLID",
			"solidFill": true,
			"associativity": false,
			"boundaryPathsCount": 1,
			"annotatedBoundary": true,
			"seedPointsCount": 0,
			"style": 0,
			"patternStyle": 1
		},
		{
			"type": "HATCH",
			"boundaries": [
				[
					{
						"x": 478259.476003918,
						"y": 6765308.44760039
					},
					{
						"x": 477697.967678746,
						"y": 6763062.4142997
					},
					{
						"x": 476534.843290891,
						"y": 6763744.24583741
					},
					{
						"x": 477256.782566112,
						"y": 6765669.417238
					},
					{
						"x": 483914.666993144,
						"y": 6766591.89520078
					},
					{
						"x": 487724.902056807,
						"y": 6765990.2791381
					},
					{
						"x": 487765.009794319,
						"y": 6761297.67384917
					},
					{
						"x": 484797.037218413,
						"y": 6764987.58570029
					},
					{
						"x": 482029.603330069,
						"y": 6765629.30950049
					},
					{
						"x": 481909.280117532,
						"y": 6762059.7208619
					},
					{
						"x": 480385.186092067,
						"y": 6759573.04113614
					},
					{
						"x": 475411.826640548,
						"y": 6758008.83937316
					},
					{
						"x": 467951.787463271,
						"y": 6760455.41136141
					},
					{
						"x": 466908.986287953,
						"y": 6764746.93927522
					},
					{
						"x": 474770.102840353,
						"y": 6761337.78158668
					},
					{
						"x": 478379.799216454,
						"y": 6760335.08814887
					},
					{
						"x": 479984.108716944,
						"y": 6762661.33692458
					},
					{
						"x": 480746.155729677,
						"y": 6764787.04701273
					},
					{
						"x": 478259.476003918,
						"y": 6765308.44760039
					}
				]
			],
			"handle": "20001",
			"layer": "0",
			"elevationX": 0,
			"elevationY": 0,
			"elevationZ": 0,
			"extrusionDirectionX": 0,
			"extrusionDirectionY": 0,
			"extrusionDirectionZ": 1,
			"patternName": "SOLID",
			"solidFill": true,
			"associativity": false,
			"boundaryPathsCount": 1,
			"annotatedBoundary": true,
			"seedPointsCount": 0,
			"style": 0,
			"patternStyle": 1
		}
	]
}